infrastructure/helm/quantumserverless/charts/gateway/templates/rayclustertemplate.yaml

```diff
@@ -20,7 +20,7 @@ data:
               # Generate head's private key and certificate before `ray start`.
 {{- if .Values.useCertManager }}
               - name: ray-head-cert
-                image: rayproject/ray:2.4.0
```

Comment:
Btw @IceKhan13 , correct me if I'm wrong but this image should not be our `ray` image as we were doing in our `client`?

- https://github.com/Qiskit-Extensions/quantum-serverless/blob/cfe62d86905dc7d46f7243238c139db3a5f8dbe5/client/quantum_serverless/core/constants.py#L32

Reply:
I think for certificates it does not matter which image to use as long as it is based on ray.
Maybe having the same image everywhere will help in managing configurations (configure it in one place instead of 2)

Reply:
I'm going to configure a reminder for this next week. For this one we had enough 😂